pkg/build/os-release.go

```diff
@@ -82,6 +82,9 @@ func (di *defaultBuildImplementation) GenerateOSRelease(
 	defer w.Close()
 
 	if ic.OSRelease.ID != "" {
+		if ic.OSRelease.ID == "unknown" {
+			o.Logger().Warnf("unidentifiable distro, may cause failure in vulnerability scanning results to correctly identify distro")
```

Comment:
```suggestion
			o.Logger().Warnf("distro ID not specified and /etc/os-release does not already exist")
```

Reply:
Thanks, improved it :))

Reply:
Let me know if it needs any other changes @kaniini, thanks a lot for the review